_data/sites.json

```
@@ -19101,6 +19101,16 @@
         "difficulty": "hard",
         "domains": ["targetprocess.com"]
     },
+    {
+        "name": "Taskade",
+        "url": "https://www.taskade.com/settings",
+        "notes": "Go to Account ⮕ Scroll down to Delete Account and Enter your Email ID",
```

```suggestion
        "notes": "Go to Account, scroll down to Delete Account and enter your Email ID",
```
I think the arrow here was not very useful